src/main/io/ledstrip.c

```diff
@@ -1246,6 +1256,24 @@ static void applySimpleProfile(timeUs_t currentTimeUs)
         switch (ledStripConfig()->ledstrip_profile) {
             case LED_PROFILE_RACE:
                 colorIndex = ledStripConfig()->ledstrip_race_color;
+#ifdef USE_VTX_COMMON
+                if (colorIndex == COLOR_BLACK) {
```

Comment:
That was the original functionality and I didn't want to change that (there must be a legacy reasons for that which I am not aware). That's why you need to disable static color to get it working automatically in RACE mode.

Reply:
> That's why you need to disable static color to get it working automatically in RACE mode.

What are the magic steps (for documentation)?

Reply:
@haslinghuis 

> What does it mean for users not using this

that's a very good question... I think there are two choices:
1. Current default is kept. Then racers and everyone else who like to use auto mode must set the `ledstrip_race_color` to `COLOR_BLACK` in RACE mode to enable this.
2. or we change default to `COLOR_BLACK` and case is vice versa :) 

I personally think the case 1 would be the best in long term if we consider that mode RACE only. Maybe it would be good idea to create a new STATIC led mode which is used for "current functionality"...